theseus/embodied/kinematics/tests/test_inverse_kinematics.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+
+import numpy as np
+import pytest
+import torch
+
+import theseus as th
+from theseus.embodied.kinematics import UrdfRobotModel
```

Comment:
By convention, in our unit tests we only import `theseus as th` and access all classes/functions in `embodied`  via `th.eb.XXXX`. See [example](https://github.com/facebookresearch/theseus/blob/main/theseus/embodied/collision/tests/test_signed_distance_field.py#L24). 

We should add `UrdfRobotModel` to [this](https://github.com/facebookresearch/theseus/blob/main/theseus/embodied/__init__.py) file and do the same here.